collection-scripts/gather_vsphere_config

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+VSPHERE_CONFIG_EVENTS_PATH=$BASE_COLLECTION_PATH/vsphere_config_events
+
+if [ -z "$(oc get deployment -n openshift-cluster-storage-operator vsphere-problem-detector-operator | grep vsphere-problem-detector-operator)" ]; then
+exit
+fi
+
+METRICS="vsphere_vcenter_info vsphere_features vsphere_cluster_check_total vsphere_cluster_check_errors vsphere_node_check_errors vsphere_node_check_total vsphere_node_hw_version_total"
+mkdir -p $VSPHERE_CONFIG_EVENTS_PATH
+for METRIC in $METRICS; do
```

Comment:
Can we collect this in asynchronously ? vs in sequence the way we are in a loop here?